.github/workflows/node.yaml

```
@@ -14,18 +14,25 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: denoland/setup-deno@v1
+      - uses: actions/setup-node@v4
         with:
-          deno-version: "1.37.2"
+          node-version: "20.x"
```

```suggestion
          node-version: 20
```
メジャーバージョンの数値のみを指定すればよいはずです